src/action.js

```diff
@@ -236,23 +247,23 @@ function markdownReport(reports, commit, options) {
         return `| ${row.filter(Boolean).join(" | ")} |`;
       })
       .join("\n");
-    const titleText = `<strong>${reportName}${folder}</strong>`;
+    const titleText = `<strong>${reportName}${folder} - ${total}%</strong>`;
     output += `${titleText}\n\n${table}\n\n`;
+    structuredOutput += `<details><summary>${titleText}</summary>\n\n${table}\n\n</details>\n\n`;
   }
   const minimumCoverageText = `_Minimum allowed coverage is \`${minimumCoverage}%\`_`;
   const footerText = `<p align="right">${credits} against ${commit} </p>`;
   output += `${minimumCoverageText}\n\n${footerText}`;
-  return output;
+  return [output, structuredOutput];
```

Comment:
the second `output` value is the one that contains the `<summary/>` block best suited for a PR comment:
<img width="555" alt="Screen Shot 2021-12-22 at 6 25 36 PM" src="https://user-images.githubusercontent.com/39174181/147173633-a3ad2578-e9fb-4f32-83a3-5eb8e467f723.png">
<img width="515" alt="Screen Shot 2021-12-22 at 6 25 45 PM" src="https://user-images.githubusercontent.com/39174181/147173636-bc7229f0-73eb-465b-9ae5-b48dd5d14baa.png">

Reply:
I could see this being an additional `input` parameter for the action, possibly called `collapse_pr_comment: bool` (or something like that)